environment/directories.rst

```diff
@@ -38,23 +34,20 @@ $HOME
 =====
 
 Your home directory is just that.
-It’s backed by essentially the same storage as your home directory on `lsst-dev`_ in the LSST Data Facility.
+
+The :file:`$HOME` directory is also the base directory for the `JupyterLab file browser`_.
+Notebooks and other files need to be inside the :file:`HOME` directory (or a subdirectory) for you to open them from the `file browser`_.
+
+Inside the :file:`$HOME` directory is a :file:`~/notebooks` directory that you can use to organize your personal notebooks.
+This directory also contains the :ref:`~/notebooks/.user_setups <lsst-kernel-user-setups>` file that you can use to configure the environment your notebooks run in.
+
+:file:`$HOME` is backed by essentially the same storage as your home directory on `lsst-dev`_ in the LSST Data Facility.
```

Comment:
Note that we will soon (or already do?) have a class of notebook users who are not staff and do not have home directories on `lsst-dev`.